### DISCUSSION TITLE
Error create DeliveryChannels - already exists

### DISCUSSION THREAD

**Discussion by BAY-GOFNX** [aws-solutions/workload-discovery-on-aws#457] [Q&A] [answered]
Hi All,

Is it possible to change the name DeliveryChannels = Default for another name??? Because I have another delivery channel with name default but it is not possible to rename because global directions... I would like to implement this solution?

Thanks

**Comment by svozza** [comment, marked as answer]
No, the delivery channel name is managed by AWS Config. If you already have Config enabled in the region you wish to deploy Workload Discovery too then you can set the `AlreadyHaveConfigSetup` CloudFormation parameter to `Yes` as detailed in the [prerequisites](https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/deployment-process-overview.html) sction of the documentation.

**Reply by BAY-GOFNX** [reply]
Thank you

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
